### PR TITLE
Update v4 tooling feed to include Durable Functions isolated templates. 

### PR DESCRIPTION
**Pull request by jviau**

### CODE REVIEW

File: cli-feed-v4.json [hunk 1]
```diff
@@ -41,12 +41,12 @@
       "hidden": false
     },
     "v4": {
-      "release": "4.30.0",
+      "release": "4.31.0",
```
Comment (jviau) [inline cli-feed-v4.json:44]: Given the templates are for a now GA product, is it okay to update both `v4` and `v4-prerelease`?

File: cli-feed-v4.json [hunk 2]
```diff
@@ -11958,6 +11958,241 @@
           "default": "false"
         }
       ]
+    },
+    "4.31.0": {
```
Comment (jviau) [inline cli-feed-v4.json:11962]: I went with a minor version rev as I think new templates qualify as a new "feature", thus following semver this is a minor version rev.

Comment (soninaren) [reply cli-feed-v4.json:11962]: Yeah that works. The only thing i would say is co-ordinate with the release team if they are doing any updates. As their changes might end up overwriting this release.

File: cli-feed-v4.json (hunk 1 above)
Comment (soninaren) [reply cli-feed-v4.json:44]: We have set up the process to let the changes bake in for a few days. VS code has some nightly tests that run on pre-release and prod tags. We have been dinged for moving directly to prod and then running into issues.

**Review by soninaren — CHANGES_REQUESTED** [review]

File: cli-feed-v4.json (hunk 1 above)
Comment (jviau) [reply cli-feed-v4.json:44]: Okay, so update prerelease only?

File: cli-feed-v4.json (hunk 2 above)
Comment (jviau) [reply cli-feed-v4.json:11962]: Who should I reach out to on the release team?

**Review by soninaren — APPROVED** [review]

**Pull request merged**